CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-
+### Added
+- `Promise.allSettled` polyfill to support NodeJS 10
+### Fixed
+-  Reporting is down on error with collect request on reporting start
```

Comment:
Use can mention the issue from Playwright repo (https://github.com/reportportal/agent-js-playwright/issues/99)